src/survey/survey.ts

```diff
@@ -25,9 +25,9 @@ export interface SurveyProps {
   cycles: {
     [key: string]: SurveyCycle
   }
-  descriptions?: Labels[] | null
+  descriptions?: Labels | null
   languages: LanguageCode[]
-  labels?: Labels | {}
+  labels?: Labels | null
```

Comment:
do we really need | null? labels is already an optional property ...

Reply:
please remove | null from labels and descriptions.. I know it comes from a previous PR..

Reply:
right, I'll remove it